workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/App/App.tsx

```diff
@@ -3,29 +3,176 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import {useParams} from 'react-router-dom';
-
-import {ReviewAndSubmitAppPage} from '../../../ReviewAndSubmitAppPage/ReviewAndSubmitAppPage';
+import {useOutletContext, useParams} from 'react-router-dom';
 
 import './App.scss';
-import useGetProductByOrderId from '../../../../hooks/useGetProductByOrderId';
+import locationIcon from '../../../../assets/icons/location_icon.svg';
+import orderFormIcon from '../../../../assets/icons/order_form_icon.svg';
+import shoppingCartIcon from '../../../../assets/icons/shopping_cart_icon.svg';
+import {DetailedCard} from '../../../../components/DetailedCard/DetailedCard';
+import {formatDate} from '../../../PublishedAppsDashboard/PublishedDashboardPageUtil';
 
 const App = () => {
 	const {orderId} = useParams();
+	const {data} = useOutletContext<any>();
+
+	const placedOrder = data.placedOrder;
+
+	const projectNameField = data.product.customFields.find(
+		(field: {name: string}) => field.name === 'Project Name'
+	);
+
+	const formatSKUName = (sku: string) => {
+		const skuLowerCased = sku.toLowerCase();
 
-	const {data} = useGetProductByOrderId(orderId);
-	const product = data?.product;
+		return skuLowerCased.charAt(0).toUpperCase() + skuLowerCased.slice(1);
+	};
```

Comment:
@claricebrasil this is used for Capitalize the text?

We could use `text-capitalize` in CSS, it's better

Reply:
Oh, good point. Thank you, Keven!